game-core/src/main/java/games/strategy/triplea/ui/history/HistoryLog.java

```diff
@@ -527,7 +527,6 @@ public void printProductionSummary(final GameData data) {
     textArea.setText(stringWriter.toString());
   }
 
-  // copied from StatPanel
```

Comment:
There is no longer a copy of this method in `StatPanel`, so this comment is no longer applicable.